### PR TITLE
POL3416 fixed no-values in ellipse and polygonality features

### PR DESCRIPTION
**Pull request by friskluft**

### CODE REVIEW

File: src/nyx/features/circle.cpp [hunk 1]
```diff
@@ -142,16 +137,16 @@ void EnclosingInscribingCircumscribingCircleFeature::findSecondPoint (const std:
     }
 }
 
-void EnclosingInscribingCircumscribingCircleFeature::findMinEnclosingCircle (const std::vector<Pixel2>& contour, Point2f& center, float& radius)
+void EnclosingInscribingCircumscribingCircleFeature::findMinEnclosingCircle (std::vector<Pixel2>& contour, Point2f& center, float& radius)
```
Comment (sameeul) [inline src/nyx/features/circle.cpp:140]: not clear why we are dropping the `const`.

File: src/nyx/features/hexagonality_polygonality.h [hunk 2]
```diff
@@ -23,5 +23,6 @@ class HexagonalityPolygonalityFeature: public FeatureMethod
 
 private:
 	double polyAve = 0, hexAve = 0, hexSd = 0;
+	static const int novalue = -1;	// feature falue made intentionally blank and unsuitable for model training
```
Comment (sameeul) [inline src/nyx/features/hexagonality_polygonality.h:26]: We need to document it somewhere, I think.

Comment (friskluft) [reply src/nyx/features/hexagonality_polygonality.h:26]: Later, when I do similar NAN intercept for other feature classes. I may have 3 more ones today

File: src/nyx/features/circle.cpp (hunk 1 above)
Comment (friskluft) [reply src/nyx/features/circle.cpp:140]: Accidentally. Reverting....

**Review by JesseMckinzie — APPROVED** [review]

**Review by sameeul — APPROVED** [review]

**Pull request merged**